packages/@lwc/engine-core/src/framework/check-version-mismatch.ts

```diff
@@ -35,15 +25,14 @@ export function checkVersionMismatch(
     type: 'template' | 'stylesheet' | 'component'
 ) {
     const versionMatcher = func.toString().match(LWC_VERSION_COMMENT_REGEX);
-    if (!isNull(versionMatcher) && !warned) {
+    if (!isNull(versionMatcher)) {
         const version = versionMatcher[1];
         const [major, minor] = version.split('.');
         const [expectedMajor, expectedMinor] = LWC_VERSION.split('.');
         if (major !== expectedMajor || minor !== expectedMinor) {
-            warned = true; // only warn once to avoid flooding the console
             // stylesheets and templates do not have user-meaningful names, but components do
             const friendlyName = type === 'component' ? `${type} ${func.name}` : type;
-            logError(
+            logErrorOnce(
```

Comment:
Since `friendlyName` will change for each component, the error will be logged for each offending component. Previously, the msg was logged once per page load. If the change in behavior was intentional, ignore this comment.

Reply:
Ah you're right. I made a mistake here.